### ISSUE TITLE
Creating new file/folder gets default name

### ISSUE THREAD

**Issue by DrPandemic** [adobe/brackets#10733]
Hi,
I'm running Cinnamon on Arch Linux x64. I just updated my Brackets to "Release 1.2 experimental build 1.2.0-0".
I think the bug was introduced in my last update. When I create a new file/folder, or rename it, the document will always be called Untitled-1, Untitled-2, etc.
Even if I reload without extension, the bug persists.

**Comment by prksingh** [comment]
Do you mean brackets doesn't popup the save dialog for new files or that after you save, the new name provided is ignored and it defaults to original name ('untitled-1' maybe?)

**Comment by DrPandemic** [comment]
You are right, that was not really clear. Brackets prompts me for a name, and when I click enter, it ignores the name and put a default one

**Comment by nethip** [comment]
@DrPandemic Could you confirm if this is happening in the released builds as well? When you say Brackets experimental builds are you referring to linux prerelease build?

**Comment by peterflynn** [comment]
@DrPandemic Between this and #10731, it sounds like you're having trouble entering getting Brackets to respond to text field input in general.

I second what @nethip said: can you confirm you're using an official, stable Brackets 1.2 build downloaded directly from http://brackets.io?  There's no build number on the end of your version number (after the "-"), so I think you have some sort of test build instead.  Can you download the official 1.2 and see if the same problem happens?

**Comment by DrPandemic** [comment]
Updating/downgrading to the official release solved the problem for me. So I guess it's not a problem currently, but it could be in the next release. Should I close this issue?
